src/sass/partials/components/_content-card.scss

```diff
@@ -0,0 +1,30 @@
+.card {
```

Comment:
I think it could be for any card, although I'm not sure what those cards would be at this point. @Snugug can comment if this is the right approach or if it should be more specific.